models/choral.js

```diff
@@ -65,6 +65,16 @@ choralSchema.statics.createNew = function (attrs, cb) {
   });
 };
 
+// get all chorals that require processing and return the result as a list
+choralSchema.statics.getAllChorals = function (cb) {
```

Comment:
maybe change to getAllChoralsWithChildren or something to indicate what the query is doing.